lib/cron/cache-pools-ipfs.ts

```diff
@@ -0,0 +1,112 @@
+import pinataSDK from '@pinata/sdk';
+import { ChainId, SubgraphProvider } from '@uniswap/smart-order-router';
+import { EventBridgeEvent, ScheduledHandler } from 'aws-lambda';
+import { Route53, STS } from 'aws-sdk';
+import { default as bunyan, default as Logger } from 'bunyan';
+import fs from 'fs';
+
+// if (process.env.STAGE == 'beta') {}
+
+const DIRECTORY = '/tmp/pools/';
+
+// add more chains here
+const chains = [ChainId.MAINNET, ChainId.RINKEBY]
+const names = ["mainnet.txt", "rinkeby.txt"]
+
```

Comment:
Nit: prefer a complex obj

`const chains: { fileName: string, chain: ChainId }[] = ..`

and later

`const { fileName, chain } = chains[i];`